libp2p/Session.cpp

```diff
@@ -332,72 +335,73 @@ void Session::doRead()
 
     auto self(shared_from_this());
     m_data.resize(h256::size);
-    ba::async_read(m_socket->ref(), boost::asio::buffer(m_data, h256::size), [this,self](boost::system::error_code ec, std::size_t length)
-    {
-        ThreadContext tc(info().id.abridged());
-        ThreadContext tc2(info().clientVersion);
-        if (!checkRead(h256::size, ec, length))
-            return;
-        else if (!m_io->authAndDecryptHeader(bytesRef(m_data.data(), length)))
-        {
-            cnetwarn << "header decrypt failed";
-            drop(BadProtocol); // todo: better error
-            return;
-        }
-        
-        uint16_t hProtocolId;
-        uint32_t hLength;
-        uint8_t hPadding;
-        try
-        {
-            RLPXFrameInfo header(bytesConstRef(m_data.data(), length));
-            hProtocolId = header.protocolId;
-            hLength = header.length;
-            hPadding = header.padding;
-        }
-        catch (std::exception const& _e)
-        {
-            cnetwarn << "Exception decoding frame header RLP: " << _e.what() << " "
-                     << bytesConstRef(m_data.data(), h128::size).cropped(3);
-            drop(BadProtocol);
-            return;
-        }
+    ba::async_read(m_socket->ref(), boost::asio::buffer(m_data, h256::size),
+        [this, self](boost::system::error_code ec, std::size_t length) {
+            LOG_SCOPED_CONTEXT(m_logContext);
```

Comment:
clang-format reformatted it a lot, but the change is just replacing `ThreadContext` declarations with `LOG_SCOPED_CONTEXT`